apps/src/fish/FishView.jsx

```diff
@@ -34,8 +34,7 @@ const styles = {
   activityCanvas: {
     width: '100%',
     borderRadius: '10px',
-    border: '2px solid black',
-    backgroundColor: 'white'
+    border: '2px solid black'
```

Comment:
backgrounds aren't in code studio yet so this might end up looking kind of weird

Reply:
Oh, we should time this with our next publish then...